docs/kop.md

```diff
@@ -0,0 +1,135 @@
+
+    ```properties
+    brokerDeleteInactiveTopicsEnabled=false
+    ```
+
+## Load KoP by restarting Pulsar brokers
+
+After you have installed the KoP protocol handler to Pulsar broker, you can restart the Pulsar brokers to load KoP if you have configured the `conf/broker.conf` file. For quick start, you can configure the `conf/standalone.conf` file and run a Pulsar standalone. You can verify if your KoP works well by running a Kafka client.
+
+1. Download [Kafka 2.0.0](https://www.apache.org/dyn/closer.cgi?path=/kafka/2.0.0/kafka_2.11-2.0.0.tgz) and untar the release package.
+
+    ```
+    tar -xzf kafka_2.11-2.0.0.tgz
+    cd kafka_2.11-2.0.0
+    ```
+
+2. Verify the KoP by using a Kafka producer and a Kafka consumer. Kafka binary contains a command-line producer and consumer.
+
+    Run the command-line producer and send messages to the server.
+
+    ```
+    > bin/kafka-console-producer.sh --broker-list [pulsar-broker-address]:9092 --topic test
+    This is a message
+    This is another message
+    ```
+
+    Kafka has a command-line consumer dumping out messages to standard output.
+
+    ```
+    > bin/kafka-console-consumer.sh --bootstrap-server [pulsar-broker-address]:9092 --topic test --from-beginning
+    This is a message
+    This is another message
+    ```
+
+# How to use KoP
+You can configure and manage KoP based on your requirements. Check the following guides for more details.
+- [Configure KoP](docs/configuration.md)
+- [Secure KoP](docs/security.md)
+- [Manage KoP with the [Envoy](https://www.envoyproxy.io) proxy](docs/envoy-proxy.md)
+- [Implementation: How to converse Pulsar and Kafka](docs/implementation.md)
+
+The following are important information when you configure and use KoP.
+
+- Set both [retention and time to live (TTL)](http://pulsar.apache.org/docs/en/cookbooks-retention-expiry/) for KoP topics. If you only configure retention without configuring TTL, all messages of KoP topics cannot be deleted because KoP does not update a durable cursor.
+-  If a Pulsar consumer and a Kafka consumer both subscribe the same topic with the same subscription (or group) name, the two consumers consume messages independently and they do not share the same subscription though the subscription name of a Pulsar client is the same with the group name of a Kafka client.
+- KoP supports interaction between Pulsar client and Kafka client by default. If your topic is used only by the Pulsar client or only by the Kafka client, you can set `entryFormat=kafka` for better performance.
+
+## Upgrade
+If you want to upgrade your KoP version, you must first upgrade your Pulsar version accordingly.
```

Comment:
1. what are the steps for upgrading a kop?

2. you can add the link of how to upgrade pulsar: https://pulsar.apache.org/docs/en/next/administration-upgrade/

Reply:
Great suggestion